pkgs/top-level/aliases.nix

```
@@ -792,6 +792,9 @@ mapAliases ({
   librsync_0_9 = throw "librsync_0_9 has been removed"; # Added 2021-07-24
   librewolf-wayland = librewolf; # Added 2022-11-15
   libseat = seatd; # Added 2021-06-24
+  libsForQt512 = throw "Qt 5 versions prior to 5.15 are no longer supported upstream";  # Added 2022-11-24
+  libsForQt514 = throw "Qt 5 versions prior to 5.15 are no longer supported upstream";  # Added 2022-11-24
```

```suggestion
  libsForQt512 = throw "Qt 5 versions prior to 5.15 are no longer supported upstream and have been removed.";  # Added 2022-11-24
  libsForQt514 = throw "Qt 5 versions prior to 5.15 are no longer supported upstream and have been removed.";  # Added 2022-11-24
```

just to make it clear why the error appears

---

Thanks, applied.